identity/Cargo.toml

```diff
@@ -31,10 +33,10 @@ harness = false
 default = ["async"]
 
 # Enables async runtime support (Tokio)
-async = ["identity-iota/async"]
+async = ["identity-iota/async", "identity-iota-core/async"]
 
 # Enables support for secure storage of DID Documents
-account = ["identity-account"]
+account = ["identity-account", "identity-account-core"]
```

Comment:
Is there no top-level `stronghold` feature for `identity`? If not we definitely need to add one.

Reply:
A `stronghold` feature flag was added in the epic branch, so it doesn't matter here.